.github/workflows/ci.yml

```diff
@@ -6,7 +6,6 @@ on:
   push:
     branches:
       - 'develop'
-      - 'v4'
```

Comment:
I know I asked this before but can't we just get rid of the `push` trigger?

Reply:
You mean so that we don't re-run the tests after merging a PR (which under normal circumstances would undergo a test run due to `pull_request` trigger)?

Reply:
Yeah, it just clogs up CI whenever multiple changes are being proposed (including in the hooks repo).

Reply:
Then often the *cancel previous jobs* job can't run because it's stuck in the queue too.